### PR TITLE
Update to Python 3.6

### PR DESCRIPTION
**Pull request by ccordoba12**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,9 +27,6 @@ test:
         - humanfriendly
     commands:
         - humanfriendly --help  # [not osx]
-    requires:
-        - capturer >=2.1  # [not win]
-        - coloredlogs >=2.0
```
Comment (ccordoba12) [inline recipe/meta.yaml:32]: This is only needed to run the test suite of `humanfriendly`. Since we're not doing this here, I decided to remove it.

**Comment by ccordoba12** [comment]
@ocefpaf, I think this is ready, please merge.

**Comment by ccordoba12** [comment]
@jakirkham, could you merge this one, please?

**Comment by andfoy** [comment]
@jakirkham @ocefpaf Could you please help us with this one?

**Comment by ccordoba12** [comment]
@andfoy, this requires now that we readd this package in staged-recipes first.

So from this PR please take the changes to `meta.yaml` (and also add you and me as maintainers) and follow this as a model PR:

https://github.com/conda-forge/staged-recipes/pull/3538

**Comment by ccordoba12** [comment]
Closing because this is useless after readding the feedstock.

**Pull request closed without merging**